Sources/WhisperKit/Core/Models.swift

```diff
@@ -321,10 +339,20 @@ public struct TranscriptionSegment: Hashable, Codable {
     public var end: Float
     public var text: String
     public var tokens: [Int]
+    public var tokenLogProbs: [[Int: Float]]
     public var temperature: Float
     public var avgLogprob: Float
     public var compressionRatio: Float
     public var noSpeechProb: Float
+    public var words: [WordTiming]?
+}
+
+public struct WordTiming: Hashable, Codable {
+    var word: String
+    var tokens: [Int]
+    var start: Float
+    var end: Float
+    var probability: Float
```

Comment:
can't actually access the timing at the moment 😅

```suggestion
    public var word: String
    public var tokens: [Int]
    public var start: Float
    public var end: Float
    public var probability: Float
```